src/Pos/Wallet/WalletMode.hs

```diff
@@ -190,10 +196,16 @@ type WalletMode ssc m
 -- Implementations of 'WalletMode'
 ---------------------------------------------------------------
 
-type WalletRealMode = PeerStateHolder
-                      (KeyStorage
-                       (WalletDB
+type FakeSsc ssc = TaggedTrans ssc IdentityT
```

Comment:
`WithNodeContext`? Wut? Why is it called `FakeSsc` then?

Also, why does lite-wallet need to satisfy constraints from full-wallet?